src/azure-cli/azure/cli/command_modules/keyvault/_client_factory.py

```diff
@@ -202,8 +182,10 @@ def data_plane_azure_keyvault_administration_access_control_client(cli_ctx, comm
 
     vault_url, credential, version = _prepare_data_plane_azure_keyvault_client(
         cli_ctx, command_args, ResourceType.DATA_KEYVAULT_ADMINISTRATION_ACCESS_CONTROL)
+    client_kwargs = prepare_client_kwargs_track2(cli_ctx)
+    client_kwargs.pop('http_logging_policy')
```

Comment:
Is this because this SDK doesn't support `http_logging_policy`?

Reply:
Keyvault data plane SDKs have fixed `http_logging_policy` and don't accept customization